CONTRIBUTING.md

```diff
@@ -23,3 +23,28 @@ If you'd like to help make RSpec better, here are some ways you can contribute:
 If you need help getting started, check out the [DEVELOPMENT](DEVELOPMENT.md) file for steps that will get you up and running.
 
 Thanks for helping us make RSpec better!
+
+## `Small` issues
+
+Small issues are the ones that we be believe are best suited for new
+contributors to get started on. They represent a meaningful contribution to the
+project that can be made that should not be too hard to pull off.
```

Comment:
This reads a bit funny to me, how about `They represent a potential meaningful contribution to the project that should not be too hard to pull off.`